### PR TITLE
fix resolution of imported css files

### PR DESCRIPTION
**Pull request by KonnorRogers**
Not sure why, but when trying to use this package with Shoelace / Web Awesome, it was not actually finding the CSS files and transforming them. The filter was never able to find them when I imported a ".css" file inside of a ".ts" file.

I released my own fork https://www.npmjs.com/package/@konnorr/esbuild-plugin-lit-css that is using the code in the PR. I figured I'd be a good user and PR the changes should you want to upstream them :)

### CODE REVIEW

**Comment by bennypowers** [comment]
Can you add a test? Were you able to reproduce?

**Comment by KonnorRogers** [comment]
I'll take a look at making a reproduction and file a separate issue. Seems this fails your tests here so maybe it doesn't work if the css file is an entry point?

**Comment by KonnorRogers** [comment]
🙃 well. Turns out the problem was a plugin.

TLDR is I had this:

```js
import { replace } from "esbuild-plugin-replace"
import litCssPlugin from "esbuild-plugin-lit-css" 

const config = {
  plugins: [
    replace(),
    litCssPlugin()
  ]
}
```

Turns out...the order of these matters. The first plugin to return `{contents: ""}` will be used by the loader and no other `onLoad` hooks will run. I'm going to close this PR as it's not an issue with the plugin. And now im even more confused why my plugin worked (perhaps because its making a namespace and dodging the replacer plugin) 🙃 

<https://github.com/evanw/esbuild/issues/2607>

<https://github.com/angular/angular-cli/issues/26326>

Reproduction: <https://github.com/KonnorRogers/esbuild-plugin-lit-css-reproduction>

**Comment by bennypowers** [comment]
Interesting! Thanks for looking into this. If we added a namespace here, would that solve the problem?

**Comment by KonnorRogers** [comment]
@bennypowers I don't believe so. What I had seemed to work by accident.
